package.json

```diff
@@ -54,9 +54,10 @@
     },
     "dependencies": {
         "axios": "^0.19.2",
+        "extract-zip": "^2.0.1",
+        "lodash": "^4.17.19",
         "rimraf": "^3.0.0",
         "unzipper": "^0.10.3",
```

Comment:
Do we have two unzipping packages here?

Reply:
correct, let me remove the previous one.

Reply:
actually I'm leaving around the previous code for future changes and that uses the unzipper so yes, we are going to ship with both.